translations/fr.json

```diff
@@ -921,14 +883,13 @@
   "credits.programming": "Programmation \u0026 Design",
   "credits.sound": "Son & Musique",
   "credits.tribe": "Design des tribus",
-  "credits.debuginfo": "Information de débugge",
+  "credits.debuginfo": "Information de débogage",
```

Comment:
The both are true in french. I think we use more "debugge" in the current language. What do you think about it?